packages/web/src/pages/settings-page/SettingsPage.tsx

```diff
@@ -2,9 +2,12 @@ import { RefObject } from 'react'
 
 import { useIsMobile } from 'hooks/useIsMobile'
 
-import SettingsPageProvider from './SettingsPageProvider'
+import { SettingsPageProvider } from './SettingsPageProvider'
```

Comment:
imo in general i feel like the more important thing to do is to remove the "provider" pattern entirely rather than make it functional. We can go forward with this PR if you'd like but in the future lets just make "SettingsPage" components without a common root that selects a bunch of state and sets up functions. make sense?

Reply:
ah yes that's better.